### DISCUSSION TITLE
Servers for crossplay

### DISCUSSION THREAD

**Discussion by henryruss2** [OhMyGuus/BetterCrewLink#59] [Q&A]
I know you can use crewlink and bettercrewlink together, but do you have to be using the same voice server?

**Comment by OhMyGuus** [comment]
Yes you do have to use the same voice server, so I recommend to use the Bettercrewlink voice server (https://bettercrewl.ink) since the default voice server for original Crewlink doesn't work anymore.  But I don't think any1 should still use original crewlink since it doesn't work on the latest version anymore
